admin/app/views/workarea/admin/content_assets/index.html.haml

```diff
@@ -77,6 +78,11 @@
                 = upcoming_changesets_icon_for(result)
                 = comments_icon_for(result)
                 = favicon_icon_for(result)
+              %td
+                - if result.image?
+                  = result.alt_text
```

Comment:
Should we be worried about really long values messing up display here?

Reply:
Yes, let's.